dev-cpp/sdbus-c++/sdbus-c++-1.4.0.ebuild

```diff
@@ -12,12 +12,15 @@ LICENSE="LGPL-2.1+ Nokia-Qt-LGPL-Exception-1.1" # Nothing to do with Qt but exce
 SLOT="0/1"
 KEYWORDS="~amd64"
 IUSE="doc +elogind systemd test tools"
-REQUIRED_USE="^^ ( elogind systemd )"
+REQUIRED_USE="?? ( elogind systemd )"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
-	elogind? ( >=sys-auth/elogind-236 )
-	systemd? ( >=sys-apps/systemd-236:= )
+	|| (
+		elogind? ( >=sys-auth/elogind-236 )
+		systemd? ( >=sys-apps/systemd-236:= )
+		>=sys-libs/basu-0.2.1
+	)
```

Comment:
I'm not sure that this logic works. If you enable `elogind` or `systemd` then I think it will allow basu _instead_ of the other libraries. It should look like this:

```
	elogind? ( >=sys-auth/elogind-236 )
	systemd? ( >=sys-apps/systemd-236:= )
	!elogind? ( !systemd? ( >=sys-libs/basu-0.2.1 ) )
```

Reply:
Oops, yeah, that is true. Fixed